.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Questions?
+    url: https://github.com/valkey-io/valkey/discussions
+    about: Please ask and answer questions here.
+  - name: Chat with us?
+    url: https://discord.gg/zbcPa5umUB
```

Comment:
There is also a matrix chat https://matrix.to/#/#valkey:matrix.org, that several folks are using.

Reply:
Thx, i'll add that too